apps/src/sites/studio/pages/levels/editors/fields/_callouts.js

```diff
@@ -0,0 +1,34 @@
+/* global levelbuilder */
+import $ from 'jquery';
+
+$(document).ready(initPage);
+
+function initPage() {
+  levelbuilder.jsonEditor('#callout_editor', {
```

Comment:
this is fine for now, but we should think about getting rid of the pattern of defining globals in levelbuilder.js in the future.